### PR TITLE
PN-10746 SP Changes

### PR DESCRIPTION
**Pull request by abhishekjir**

### CODE REVIEW

**Comment by projectpas** [comment]
@abhishekjir , dbo and nolock was missing

File: PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql [hunk 1]
```diff
@@ -71,9 +73,18 @@ BEGIN
 		SELECT @NetBookValue = (ISNULL(ISNULL(@InstalledCost,0) - ISNULL(@AccumlatedDepr,0),0));												-- @InstalledCost - @AccumlatedDepr;
 		SELECT @NBVAfterDepreciation = (ISNULL(ISNULL(@NetBookValue,0) - ISNULL(@DepreciationAmount,0),0));										-- @NetBookValue - @DepreciationAmount;
 		
-		SELECT @LastDeprRunPeriod = PeriodName FROM AccountingCalendar WHERE AccountingCalendarId = @SelectedAccountingPeriodId 
+		SELECT @LastDeprRunPeriod = PeriodName FROM AccountingCalendar WITH(NOLOCK) WHERE AccountingCalendarId = @SelectedAccountingPeriodId
```
Comment (amitghediyasilverxis) [inline PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql]: DBO missing

File: PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql [hunk 2]
```diff
@@ -60,9 +61,10 @@ BEGIN
 		DECLARE @Hundred DECIMAL(18,2) = 100;
 		DECLARE @DepreciationStartDate DATETIME;
 		DECLARE @LastDeprRunPeriod VARCHAR(30)
+		DECLARE @FullyDepreciatedStatusId BIGINT; 
 
-		SELECT @AccumlatedDepr = [AccumlatedDepr] FROM AssetDepreciationHistory WHERE [AssetInventoryId] = @AssetInventoryId AND [ID] = (SELECT MAX(ID) FROM AssetDepreciationHistory WHERE [AssetInventoryId] = @AssetInventoryId)
-		SELECT @ResidualPercentage = ResidualPercentage FROM AssetInventory WHERE [AssetInventoryId] = @AssetInventoryId 
+		SELECT @AccumlatedDepr = [AccumlatedDepr] FROM AssetDepreciationHistory WHERE [AssetInventoryId] = @AssetInventoryId AND [ID] = (SELECT MAX(ID) FROM AssetDepreciationHistory WITH(NOLOCK) WHERE [AssetInventoryId] = @AssetInventoryId)
+		SELECT @ResidualPercentage = ResidualPercentage FROM AssetInventory WITH(NOLOCK) WHERE [AssetInventoryId] = @AssetInventoryId
```
Comment (amitghediyasilverxis) [inline PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql]: DBO missing

File: PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql [hunk 3]
```diff
@@ -60,9 +61,10 @@ BEGIN
 		DECLARE @Hundred DECIMAL(18,2) = 100;
 		DECLARE @DepreciationStartDate DATETIME;
 		DECLARE @LastDeprRunPeriod VARCHAR(30)
+		DECLARE @FullyDepreciatedStatusId BIGINT; 
 
-		SELECT @AccumlatedDepr = [AccumlatedDepr] FROM AssetDepreciationHistory WHERE [AssetInventoryId] = @AssetInventoryId AND [ID] = (SELECT MAX(ID) FROM AssetDepreciationHistory WHERE [AssetInventoryId] = @AssetInventoryId)
-		SELECT @ResidualPercentage = ResidualPercentage FROM AssetInventory WHERE [AssetInventoryId] = @AssetInventoryId 
+		SELECT @AccumlatedDepr = [AccumlatedDepr] FROM AssetDepreciationHistory WHERE [AssetInventoryId] = @AssetInventoryId AND [ID] = (SELECT MAX(ID) FROM AssetDepreciationHistory WITH(NOLOCK) WHERE [AssetInventoryId] = @AssetInventoryId)
```
Comment (amitghediyasilverxis) [inline PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql]: DBO missing & NOLOCK

File: PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql (hunk 1 above)
Comment (abhishekjir) [reply PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql]: DBO added

File: PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql (hunk 3 above)
Comment (abhishekjir) [reply PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql]: DBO & NOLOCK added

File: PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql (hunk 2 above)
Comment (abhishekjir) [reply PAS_Database/dbo/Stored Procedures/SaveAssetDeprciationData.sql]: DBO added

**Pull request merged**